File: PRACTICE/MODULES/C++/daily_practice/13-09-2024/Association/Address.cpp
#include<iostream>
#include<string>
using namespace std;

class Address {
	private:
		string houseNo;
		string area;
		string pinNo;
	public:
		Address() : houseNo(""), area(""), pinNo("") {}
		Address(string houseNo, string area, string pinNo) : houseNo(houseNo), area(area), pinNo(pinNo) {}
		void Accept() {
			cout<<"enter Address Data : "<<endl;
			cout<<"house no is : "<<endl;
			cin>>houseNo;
			cout<<"house area is : "<<endl;
			cin>>area;
			cout<<"house pin no is : "<<endl;
			cin>>pinNo;
		}
		void Display() {
			cout<<" House Data is : "<<endl;
			cout<<" house no : "<<houseNo<<endl;
			cout<<" house area : "<<area<<endl;
			cout<<" house pin no is : "<<pinNo<<endl;
		}
};

File: PRACTICE/MODULES/OOPsWithJava/DailyPractice/JavaEclipse/Sept27/src/com/entity/Address.java
package com.entity;

import java.util.Scanner;

public class Address {
	private String houseNo;
	private String appartment;
	private String area;
	public Address() {
	
	}
	public Address(String houseNo, String appartment, String area) {
		this.houseNo = houseNo;
		this.appartment = appartment;
		this.area = area;
	}
	public String getHouseNo() {
		return houseNo;
	}
	public void setHouseNo(String houseNo) {
		this.houseNo = houseNo;
	}
	public String getAppartment() {
		return appartment;
	}
	public void setAppartment(String appartment) {
		this.appartment = appartment;
	}
	public String getArea() {
		return area;
	}
	public void setArea(String area) {
		this.area = area;
	}
	public void AAccept(Scanner sc) {
		System.out.println("enter House Number");
		setHouseNo(sc.nextLine());
		System.out.println("enter Appartment Name");
		setAppartment(sc.nextLine());
		System.out.println("enter Area");
		setArea(sc.nextLine());
	}
	@Override
	public String toString() {
		return "Address [houseNo=" + houseNo + ", appartment=" + appartment + ", area=" + area + "]";
	}
	
}
